tools/setup_permissions.sh

```diff
@@ -46,6 +46,15 @@ function grant_task_execution() {
          -H "Authorization: token=${ACS_TOKEN}" \
          "${DCOS_URL}/acs/api/v1/acls/dcos:mesos:master:task:user:${LINUX_USER}/users/${SERVICE_ACCOUNT_NAME}/create"
 
+    # XXX 1.10 curerrently requires this mesos:agent permission as well as
+    # mesos:task permission.  unclear if this will be ongoing requirement.
+    # See DCOS-15682
+    curl -k -L -X PUT \
+         -H "Authorization: token=${ACS_TOKEN}" \
+         "${DCOS_URL}/acs/api/v1/acls/dcos:mesos:agent:task:user:${LINUX_USER}" \
```

Comment:
Reading through the implementation of the code behind the ACL crap, it seems there is the possibility to create an ACL in abstract that applies to *nothing*.  (Perhaps it applies to everything?  Who knows.)

Thus the call I added creates this new ACL (if it doesn't already exist).  An ACL that says that tasks can be run as the user in question.  However, this ACL isn't applying to the service account specifically, as was done in the prior lines.  Empirically, this gave a sitaution where frameworks worked.  However, I have no idea how the permission system actually works, since there seem to be no real docs of any kind so I should at least keep in parallel with what went before.